<?xml version="1.0"?>

<launch>

  <node pkg="pointcloud2_to_object_vec_converter" name="PointCloud2ToObjectVecConverterServer" type="pointcloud2_to_object_vec_converter_node" output="screen">
  </node>

</launch>
